### PR TITLE
Fixes not including app.afterResponse functions in requests

### PR DESCRIPTION
**Pull request by BrunoBernardino**

### CODE REVIEW

File: src/tools/create-app-request-client.js [hunk 1]
```diff
@@ -41,18 +41,18 @@ const createAppRequestClient = (input, options) => {
...
 
   if (app.authentication) {
     if (app.authentication.type === 'oauth2' && _.get(app, 'authentication.oauth2Config.autoRefresh')) {
-      httpAfters.push(throwForStaleAuth);
+      httpOriginalAfters.push(throwForStaleAuth);
     }
   }
 
-  httpAfters.concat(ensureArray(app.afterResponse));
+  const httpAfters = httpOriginalAfters.concat(ensureArray(app.afterResponse));
```
Comment (BrunoBernardino) [inline src/tools/create-app-request-client.js]: This was really the culprit.

File: src/tools/create-app-request-client.js [hunk 2]
```diff
@@ -41,18 +41,18 @@ const createAppRequestClient = (input, options) => {
...
   ];
 
   if (app.authentication) {
     if (app.authentication.type === 'oauth2' && _.get(app, 'authentication.oauth2Config.autoRefresh')) {
-      httpAfters.push(throwForStaleAuth);
+      httpOriginalAfters.push(throwForStaleAuth);
     }
   }
 
-  httpAfters.concat(ensureArray(app.afterResponse));
```
Comment (BrunoBernardino) [inline src/tools/create-app-request-client.js:55]: This was really the culprit.

Comment (ibolmo) [reply src/tools/create-app-request-client.js:55]: ah yep doesn't modify the httpAfters.. fyi could also `httpAfters.push.apply(httpAfters, ensureArray(app.afterResponse));` but yeah this is cleaner :D

**Review by ibolmo — APPROVED** [review]
Just tested on my repo, and LGTM.

**Pull request merged**